admin/src/app/Components/hotels-filter/hotels-filter.component.html

```diff
@@ -0,0 +1 @@
+<aside></aside>
```

Comment:
Why do you need it?

Reply:
there is some location logics for this element in css